src/overlays/gamestates/ovl_file_choose/z_file_choose_NES.c

```diff
@@ -4,96 +4,2362 @@
+    FileSelect_UpdateMainMenu, // CM_MAIN_MENU
+    // Copy File
+    FileSelect_SetupCopySource,      // CM_SETUP_COPY_SOURCE
+    FileSelect_SelectCopySource,     // CM_SELECT_COPY_SOURCE
+    FileSelect_SetupCopyDest1,       // CM_SETUP_COPY_DEST_1
+    FileSelect_SetupCopyDest2,       // CM_SETUP_COPY_DEST_2
+    FileSelect_SelectCopyDest,       // CM_SELECT_COPY_DEST
+    FileSelect_ExitToCopySource1,    // CM_EXIT_TO_COPY_SOURCE_1
+    FileSelect_ExitToCopySource2,    // CM_EXIT_TO_COPY_SOURCE_2
+    FileSelect_SetupCopyConfirm1,    // CM_SETUP_COPY_CONFIRM_1
+    FileSelect_SetupCopyConfirm2,    // CM_SETUP_COPY_CONFIRM_2
+    FileSelect_CopyConfirm,          // CM_COPY_CONFIRM
+    FileSelect_CopyWaitForFlashSave, // CM_COPY_WAIT_FOR_FLASH_SAVE
+    FileSelect_ReturnToCopyDest,     // CM_RETURN_TO_COPY_DEST
+    FileSelect_CopyAnim1,            // CM_COPY_ANIM_1
+    FileSelect_CopyAnim2,            // CM_COPY_ANIM_2
+    FileSelect_CopyAnim3,            // CM_COPY_ANIM_3
+    FileSelect_CopyAnim4,            // CM_COPY_ANIM_4
+    FileSelect_CopyAnim5,            // CM_COPY_ANIM_5
+    FileSelect_ExitCopyToMain,       // CM_COPY_RETURN_MAIN
+    // Erase File
+    FileSelect_SetupEraseSelect,      // CM_SETUP_ERASE_SELECT
+    FileSelect_EraseSelect,           // CM_ERASE_SELECT
+    FileSelect_SetupEraseConfirm1,    // CM_SETUP_ERASE_CONFIRM_1
+    FileSelect_SetupEraseConfirm2,    // CM_SETUP_ERASE_CONFIRM_2
+    FileSelect_EraseConfirm,          // CM_ERASE_CONFIRM
+    FileSelect_ExitToEraseSelect1,    // CM_EXIT_TO_ERASE_SELECT_1
+    FileSelect_ExitToEraseSelect2,    // CM_EXIT_TO_ERASE_SELECT_2
+    FileSelect_EraseAnim1,            // CM_ERASE_ANIM_1
+    FileSelect_EraseWaitForFlashSave, // CM_ERASE_WAIT_FOR_FLASH_SAVE
+    FileSelect_EraseAnim2,            // CM_ERASE_ANIM_2
+    FileSelect_EraseAnim3,            // CM_ERASE_ANIM_3
+    FileSelect_ExitEraseToMain,       // CM_EXIT_ERASE_TO_MAIN
+    FileSelect_UnusedCM31,            // CM_UNUSED_31
+    // New File Name Entry
+    FileSelect_RotateToNameEntry,         // CM_ROTATE_TO_NAME_ENTRY
+    FileSelect_StartNameEntry,            // CM_START_NAME_ENTRY
+    FileSelect_UpdateKeyboardCursor,      // CM_NAME_ENTRY
+    FileSelect_NameEntryWaitForFlashSave, // CM_NAME_ENTRY_WAIT_FOR_FLASH_SAVE
+    FileSelect_RotateToMain,              // CM_NAME_ENTRY_TO_MAIN
+    // Options
+    FileSelect_RotateToOptions,         // CM_MAIN_TO_OPTIONS
+    FileSelect_StartOptions,            // CM_START_OPTIONS
+    FileSelect_UpdateOptionsMenu,       // CM_OPTIONS_MENU
+    FileSelect_OptionsWaitForFlashSave, // CM_OPTIONS_WAIT_FOR_FLASH_SAVE
+    FileSelect_RotateToMain,            // CM_OPTIONS_TO_MAIN
+    // Possible Debug
+    FileSelect_UnusedCMDelay, // CM_UNUSED_DELAY
+};
+
+s16 sCursorAlphaTargets[] = { 70, 200 };
+
+/**
+ * Updates the alpha of the cursor to make it pulsate.
+ */
+void FileSelect_PulsateCursor(GameState* thisx) {
```

Comment:
"Pulsate" is intransitive in most of its standard meanings, only
> 3. transitive. Mining. To vibrate (diamond-bearing material) with a pulsator (pulsator n. 3b). rare.

is given in the OED. "pulse" has transitive meanings roughly like what this is supposed to mean, although maybe `VaryCursorBrightness` or `VaryCursorGlow` would be better anyway.

Reply:
Function names aren't designed to be grammatically correct. The cursor is the subject, not the direct object. "The cursor pulsates" captured by the function `FileSelect_PulsateCursor` seems fine to me?

Reply:
I don't understand that argument, English uses SVO for this sort of construction so by that logic it should be `CursorPulsate`, if you think the cursor is the subject?